examples/ipc_server_with_database.rs

```diff
@@ -69,6 +69,7 @@ fn run_webserver(db_socket_path: &str) {
     // extrasafe context
     SafetyContext::new()
         .enable(Networking::nothing()
+            .allow_connect().yes_really()
```

Comment:
Do the tests here actually fail without the allow_connect() here? A TCP server shouldn't need to use connect AFAIK